.github/actions/gmt-pytest/action.yml

```diff
@@ -44,6 +44,7 @@ runs:
       run: |
         source venv/bin/activate
         python3 -m pip install -r requirements-dev.txt
+        playwright install --with-deps firefox
```

Comment:
In the description you write with Chrome and here you also add Firefox

Reply:
thanks. description is wrong. Firefox is what I want